okd/src/build.sh

```diff
@@ -0,0 +1,41 @@
+#/bin/bash
+
+
+# replace microshift assets to Upsteam from OKD
+okd_url=quay.io/okd/scos-release
+okd_releaseTag=4.17.0-0.okd-scos-2024-08-21-100712
+
+oc adm release info ${okd_url}:${okd_releaseTag} >all_images
```

Comment:
Is this step only to check if we are able to get the release info for the okd release or `all_images` file is used somewhere?

Reply:
no its not used, it was for debugging, so i will remove it.